cluster/gke/config-test.sh

```diff
@@ -20,6 +20,7 @@ NETWORK=${KUBE_GKE_NETWORK:-e2e}
 NODE_TAG="k8s-${CLUSTER_NAME}-node"
 IMAGE_TYPE="${KUBE_GKE_IMAGE_TYPE:-container_vm}"
 ENABLE_KUBERNETES_ALPHA="${KUBE_GKE_ENABLE_KUBERNETES_ALPHA:-}"
+NODE_OS_DISTRIBUTION=gci
```

Comment:
ok, agreeing with @msau42 here that probably `NODE_OS_DISTRIBUTION` should match `IMAGE_TYPE`.

confusingly, `gcloud container get-server-config` doesn't list `gci` as a valid image type, yet that's what we use in all of our job configs for `KUBE_GKE_IMAGE_TYPE`?
```console
$ gcloud container get-server-config --zone us-central1-f --format='value(validImageTypes)'
Fetching server config for us-central1-f
COS;CONTAINER_VM
```